pkg/server/k8s/client.go

```diff
@@ -312,16 +311,133 @@ func (k *k8sClient) Limits(namespace, name string) (*app.Limits, error) {
 	return lim, nil
 }
 
-func newInClusterK8sClient() (Client, error) {
-	conf, err := restclient.InClusterConfig()
+func (k *k8sClient) CreateDeploy(deploySpec *deploy.DeploySpec) error {
+	replicas := k.currentPodReplicasFromDeploy(deploySpec.Namespace, deploySpec.Name)
```

Comment:
I don't like functions with more than one proposal but you're right, the current name does not reflect the reality.